sites/templates/sites/leaderboard.html

```diff
@@ -45,7 +45,7 @@ <h5 class="page-subhead">See which news publications have the most secure HTTPS
   </section>
 
   <!-- Dump the site data to be rendered client-side. -->
-  <script type="text/javascript">
+  <script type="text/javascript" >
```

Comment:
Nitpick: extra space added here is probably not needed.

Reply:
Done